{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -133,13 +128,87 @@ accept-no-yields-doc = false
 default-docstring-type = "google"
 
 [tool.pylint.design]
-max-locals=20
+max-locals = 20
 
 [tool.pylint.similarities]
-min-similarity-lines=10
+min-similarity-lines = 10
 
 [tool.bandit]
-exclude_dirs = ["venv",]
+
+
+
+
+{% if cookiecutter.versioning == "Python Semantic Release" %}
+exclude_dirs = ["venv"]
 # B101 disables errors for asserts in the code
 # remember to not use asserts for security and control flows
-skips = ["B101"]
\ No newline at end of file
+skips = ["B101"]
```

Comment:
Exclude_dir and skips shall be independent of  "cookiecutter.versioning"

Reply:
typo, fixed